test/test_torch.py

```diff
@@ -1940,6 +1940,32 @@ def test_fmod(self):
             res2[i, 3] = math.fmod(res2[i, 3], q)
         self.assertEqual(res1, res2)
 
+        def _test_fmod_with_size(size, device, is_tensor=True):
+            if is_tensor:
+                a = torch.rand(size=size, device=device, dtype=torch.double)
+                b = torch.rand(size=size, device=device, dtype=torch.double)
+                actual = a.fmod(b)
+                x = a.view(-1)
+                y = b.view(-1)
+                expected = torch.tensor([math.fmod(x[i].item(), y[i].item()) for i in range(x.numel())],
+                                        device=device, dtype=torch.double)
+                self.assertTrue(torch.allclose(expected, actual.view(-1), rtol=0, atol=0.02))
+            else:
```

Comment:
Much cleaner to have `_test_fmod_with_size_tensor` and `_test_fmod_with_size_scalar`

In this case
```python
for device in torch.testing.get_all_device_types():
            _test_fmod_with_size((2, 2), device)
            _test_fmod_with_size((3, 3), device)
           # ...
```

can turn into

```python
for device in torch.testing.get_all_device_types():
	for size in [(2,2),(3,3),(5,5)]:
	_test_fmod_with_size_tensor(size, device)
	_test_fmod_with_size_scalar(size, device)
```

Reply:
Fixed